.changes/v2.16.0/485-improvements.md

```diff
@@ -0,0 +1 @@
+* Add `SupportedFeatureSet` attribute to `NsxtAlbServiceEngineGroup` and `NsxtAlbConfig` to support v37.0 license management for AVI Load Balancer [GH-485]
```

Comment:
Should I add `LicenseType` to deprecations section? It is still present in the used structs and by the API in the newest VCD, although seems ignored, so that's why I didn't.

Reply:
I think it is not worth deprecating `LicenseType` yet. 
The point is - all currently supported VCD versions use it and it is correct usage from that point. Marking this field as deprecated in our schema would tell that this field is deprecated, but there is no replacement (until one goes beyond VCD 10.3). Let's see how it goes, but IMO deprecation should happen later on.

Reply:
Just an extra sentence that `SupportedFeatureSet` field in fact replaces `LicenseType` in V37.0. That one - whoever is skimming changelog should have enough information to get triggered about it (or not).